modules/builds-configuration-file.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * builds/build-configuration.adoc
+
+[id="builds-configuration-file_{context}"]
+= Configuring build settings
+
+You can configure build settings by editing the
+`build.config.openshift.io/cluster` resource.
+
+.Procedure
+
+* Edit the `build.config.openshift.io/cluster` resource:
++
+----
+$ oc edit build.config.openshift.io/cluster
+----
++
+The following is an example `build.config.openshift.io/cluster` resource:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Build<1>
+metadata:
+  annotations:
+    release.openshift.io/create-only: "true"
+  creationTimestamp: "2019-05-17T13:44:26Z"
+  generation: 2
+  name: cluster
+  resourceVersion: "107233"
+  selfLink: /apis/config.openshift.io/v1/builds/cluster
+  uid: e2e9cc14-78a9-11e9-b92b-06d6c7da38dc
+spec:
+  additionalTrustedCA:<2>
+    name: myconfigmap
```

Comment:
@bmcelvee `additionalTrustedCA` should be deprecated and not referenced in the official docs. It doesn't do anything in 4.1.

Adding CAs should be done via `image.config.openshift.io/cluster`